### ISSUE TITLE
how to use pretrained model

### ISSUE THREAD

**Issue by JoFebriann** [Kartik-3004/hyp-oc#2]
i have take a look from your HuggingFace (https://huggingface.co/kartiknarayan/hyp-oc) and i see that you are provide the pretrained_models and weights, but how can i use it? I want to test the model without training, sorry i am very new with this topic, so may someone help me? And i also see that this hyp-oc is kinda new, so there also not many code provided on the internet. I have no idea how to use the pretrained model and test the model accuracy for face anti spoofing, could someone please help me? Thankyouu.

**Comment by Kartik-3004** [comment]
Download the weights as described [here](https://github.com/Kartik-3004/hyp-oc#download-pre-trained-weights). Verify the directory structure of the downloaded weights. 

The follow the inference script written in [README.md](https://github.com/Kartik-3004/hyp-oc/tree/main?tab=readme-ov-file#inference)

```
python test.py \
    --source_dataset ReplayAttack \
    --target_dataset MSU_MFSD \
    --device 0 \
    --curvature 0.1 \
    --batch_size_test 32 \
    --list replayattack
```
